packages/vuetify/src/components/VOverlay/useActivator.tsx

```diff
@@ -78,7 +161,9 @@ function _useActivator (props: ActivatorProps, { activatorEl, onActivatorClick }
   function bindActivatorProps (el = getActivator(), _props = props.activatorProps) {
     if (!el) return
 
-    el.addEventListener('click', onActivatorClick)
+    Object.entries(activatorEvents.value).forEach(([name, cb]) => {
```

Comment:
Could this be a for...loop for consistency purposes?

Reply:
consistency with what